crates/cli/Cargo.toml

```
@@ -93,6 +100,9 @@ parking_lot.workspace = true
 pretty_assertions.workspace = true
 serial_test.workspace = true
 
+[build-dependencies]
+chrono = "0.4.26"
```

An added dependency, but one that's already used elsewhere in the repo, so not completely new

---

```suggestion
chrono.workspace = true
```
This helps keep all the versions in sync :)